_pytest/cacheprovider.py

```diff
@@ -179,6 +216,10 @@ def pytest_addoption(parser):
         help="run all tests but run the last failures first.  "
              "This may re-order tests and thus lead to "
              "repeated fixture setup/teardown")
+    group.addoption(
+        '--nf', '--new-first', action='store_true', dest="newfirst",
+        help="run all tests but run new tests first, then tests from "
```

Comment:
I think this reads better: `run tests from new files first, then the rest of the tests sorted by file mtime`